Feature/newsletter template #402

Malwurf opened the pull request:
## Changes
closes #402 
<!-- Describe the changes which you did and which issue you're closing
 example: closes #230
-->




<!-- Paste here screenshot if there are visual changes -->





### Checklist

- [x] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

akrajna reviewed (changes requested):
- after entering the correct email address, the message '' appears

Malwurf commented:
I've fixed **storefrontUrt: The value you selected is not a valid choice** error display.

It can't be tested on the Canary environment until merge to master.

It is thank you message after the success subscription:
![Screenshot from 2020-08-26 13-09-36](https://user-images.githubusercontent.com/4071200/91307085-c80ee800-e7ad-11ea-80e1-3298e2c42bf0.png) and we have another task to make it prettier here -> #1052 

And removed -> 'Please fill form data' appears. information

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/DivanteLtd/shopware-pwa?pullRequest=1039) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/DivanteLtd/shopware-pwa?pullRequest=1039) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/DivanteLtd/shopware-pwa?pullRequest=1039) it.</sub>

Malwurf commented:
Newsletter bar before sends on desktop:
![Screenshot from 2020-08-28 15-15-02](https://user-images.githubusercontent.com/4071200/91564846-8b73f580-e941-11ea-847e-a57ccbc83750.png)
![Screenshot from 2020-08-28 15-14-35](https://user-images.githubusercontent.com/4071200/91564850-8ca52280-e941-11ea-835f-071c73ea9150.png)

After sending on desktop:
![Screenshot from 2020-08-28 15-15-11](https://user-images.githubusercontent.com/4071200/91564890-9b8bd500-e941-11ea-9576-45474179fa57.png)

Newsletter bar before sends on mobile:
![Screenshot from 2020-08-28 15-15-50](https://user-images.githubusercontent.com/4071200/91564929-ab0b1e00-e941-11ea-8996-b267ae41f88d.png)
![Screenshot from 2020-08-28 15-15-36](https://user-images.githubusercontent.com/4071200/91564930-aba3b480-e941-11ea-8402-99a5990f1b8d.png)

After sending on mobile:
![Screenshot from 2020-08-28 15-15-26](https://user-images.githubusercontent.com/4071200/91564956-b65e4980-e941-11ea-97a8-7dd92d55eabf.png)
